### PR TITLE
Created Cart management page

### PR DESCRIPTION
**Pull request by anushasunil**
image

### CODE REVIEW

File: Cart-management/cart-management.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            </div>
+        </div>
+        <footer class=" display-justify-space-around display-align-center">
+            <div class="brand-info">
+
+                <h6>BlueBasket</h6>
+                <p>Copyright © 2022</p>
+                <p>All rights reserved</p>
+                <div class="social-links display-align-center">
+                   <a href="#" class="links"><i class="fa-brands fa-github"></i></a>
```
Comment (coldpigli) [inline Cart-management/cart-management.html]: Suggestion to add social links.

File: Cart-management/cart-management.css [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    flex-wrap: wrap;
+}
+
+footer h6 {
+    padding: 1rem 0;
+}
+
+li {
+    list-style: none;
+}
```
Comment (dev-enforced) [inline Cart-management/cart-management.css]: Can store this piece of code in utilities.css if it is used frequently

**Review by coldpigli — APPROVED** [review]
Code is good to merge. Everything is well structured.

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!! The demonstration from image also looks clean!!
Good work @anushasunil

File: Cart-management/cart-management.css [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    padding: 1rem;
+}
+
+.input-box button {
+    padding: 1.2rem;
+    outline: none;
+}
+
+button {
+    border-radius: 0;
```
Comment (rittik-ghosh) [inline Cart-management/cart-management.css:20]: The border radius is by default zero

File: Cart-management/cart-management.css [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+.card .image-container {
+    width: 40%;
+}
+
+.image-container {
+    width: fit-content;
+}
+
+.star-img {
+    color: orange;
```
Comment (rittik-ghosh) [inline Cart-management/cart-management.css]: can be put in a variable

**Review by rittik-ghosh — COMMENTED** [review]
Have suggested a few changes. Overall the code is neat, clean and readable.

File: Cart-management/cart-management.css (hunk 3 above)
Comment (anushasunil) [reply Cart-management/cart-management.css:20]: yes I agree but the component is taken from the component library and there I have initialized some border-radius so here it is nullifying the effect.

File: Cart-management/cart-management.css (hunk 4 above)
Comment (anushasunil) [reply Cart-management/cart-management.css]: yes, I'll rectify it

File: Cart-management/cart-management.css (hunk 2 above)
Comment (anushasunil) [reply Cart-management/cart-management.css]: yes, I'll do that

File: Cart-management/cart-management.html (hunk 1 above)
Comment (anushasunil) [reply Cart-management/cart-management.html]: Thanks for pointing this out, will rectify it

**Pull request merged**